.github/workflows/poll_aqi.yml

```diff
@@ -0,0 +1,13 @@
+name: Poll /api/forecast
```

Comment:
This is a good idea! Couldn't you query the /aqi endpoint with this instead of the /forecast one and remove the aqi from the /forecast endpoint entirely?